core/services/relay/relay.go

```diff
@@ -28,6 +31,15 @@ var SupportedNetworks = map[string]struct{}{
 	NetworkDummy: {},
 }
 
+// From returns a NetworkType from a string.
+func From(s string) (NetworkType, error) {
+	s = strings.ToLower(s)
+	if _, ok := SupportedNetworks[s]; !ok {
```

Comment:
What is the value add here? This will be too strict in the future when not all chains are compiled in, and will have to be removed. It may be better to leave it out.

Reply:
Thanks, i have now removed the network enum!